packages/travix-ui-kit/components/stepper/stepper.js

```diff
@@ -0,0 +1,135 @@
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import React, { Component } from 'react';
+
+/**
+ * General Stepper component
+ */
+class Stepper extends Component {
+  constructor(props) {
+    super(props);
+
+    this.validationFn = (stepper, triedValue) => stepper.setValue(triedValue);
+    this.state = {
+      currentValue: Math.max(Math.min(props.initValue, props.maxValue), props.minValue),
+    };
+
+    if (typeof props.validationFn === 'function') {
+      this.validationFn = props.validationFn;
```

Comment:
I don't think you should allow users to pass the validation function as a parameter.

minValue and maxValue are more than enough.

And the external function will not work because it needs access to the internal instance to be able to setState.